projects/ngx-hateoas-client/src/lib/model/resource/abstract-resource.ts

```diff
@@ -31,4 +31,18 @@ export abstract class AbstractResource {
     return relationLink;
   }
 
+  /**
+   * Checks if relation link is present.
+   *
+   * @param relationName used to check for the specified relation name
+   * @returns true if link is present, false otherwise
+   */
+  public hasRelation(relationName: string): boolean {
+    if (isEmpty(this._links)) {
+      return false;
+    } else {
+      return Object.keys(this._links).indexOf(relationName) !== -1;
```

Comment:
For performance, it would be better to try to get relation from the links array by index without iteration over it. Something like this `return !isEmpty(this._links[relationName])`